fluentd/configs.d/openshift/filter-syslog-record-transform.conf

```diff
@@ -37,6 +37,7 @@
     # Plus we'd better avoid the file IO.
     hostname ${_HOSTNAME.eql?('localhost') ? (begin; File.open('/etc/docker-hostname') { |f| f.readline }.rstrip; rescue; _HOSTNAME; end) : _HOSTNAME}
     message ${record["MESSAGE"]}
+    level ${record["PRIORITY"]}
```

Comment:
@portante ok.  Then the problem is that it isn't clear how to translate PRIORITY numeric values to level string values.  See https://github.com/ViaQ/elasticsearch-templates/issues/35